jsonschema2pojo-ant/src/main/java/org/jsonschema2pojo/ant/Jsonschema2PojoTask.java

```diff
@@ -291,19 +291,19 @@ public void setIncludeConstructors(boolean includeConstructors) {
     }
 
     /**
-     * Sets the 'constructorsRequiredPropertiesOnly' property of this class. This is a legacy configuration option used to turn on the {@link
-     * #isIncludeAllPropertiesConstructor()} and off the {@link #isIncludeAllPropertiesConstructor()} configuration options. It
-     * is specifically tied to the {@link #isIncludeConstructors()} property, and will do nothing if that property is not enabled
+     * Sets the 'constructorsRequiredPropertiesOnly' property of this class.
+     * This is a legacy configuration option used to turn on the {@link #isIncludeRequiredPropertiesConstructor()}
+     * and off the {@link #isIncludeAllPropertiesConstructor()} configuration options.
+     * It is specifically tied to the {@link #isIncludeConstructors()} property, and will do nothing if that property is not enabled
      *
-     * @param constructorsRequiredPropertiesOnly controls whether the resulting class will have only the constructor with required properties or
-     * something else
+     * @param constructorsRequiredPropertiesOnly controls whether the resulting class will have only the constructor with required or all properties
      */
     public void setConstructorsRequiredPropertiesOnly(boolean constructorsRequiredPropertiesOnly) {
         this.constructorsRequiredPropertiesOnly = constructorsRequiredPropertiesOnly;
     }
 
     /**
-     * Sets the 'constructorsIncludeRequiredPropertiesConstructor' configuration option. This property works in collaboration with the {@link
```

Comment:
Looks like The `'constructorsIncludeRequiredPropertiesConstructor' configuration option ...` has been copy-pasted several times.
Replaced constructorsIncludeRequiredPropertiesConstructor with actual property that the Javadoc is for (also in `GenerationConfig` and `JsonSchema2PojoMojo`)